tools/changelog/README.md

```diff
@@ -0,0 +1,54 @@
+# Generating repository changelogs
+
+This script uses the developer's clone of the repository to generate a changelog via `git log`.
+
+## Requirements:
+- Node.js > 10.0.0
+- Current clone of https://github.com/microsoft/botframework-sdk
+- Current clone of the repository and branch to generate a changelog from
+
+## Preferred setup:
+- **Each repository should only permit squash merges**, otherwise the developer calling the script will need to manually update entries.
```

Comment:
Might be good to add a final step, pointing out this will most likely be necessary regardless:
 _developer calling the script will need to manually update entries._

Some commit messages are not very useful or descriptive

Reply:
Done!